config/config.go

```diff
@@ -1489,6 +1490,8 @@ func SetClientDefaults(v *viper.Viper) error {
 	configDir := v.GetString("ConfigDir")
 
 	v.SetDefault(param.IssuerKey.GetName(), filepath.Join(configDir, "issuer.jwk"))
+	v.SetDefault(param.IssuerKeysDirectory.GetName(), filepath.Join(configDir, "issuer-keys"))
```

Comment:
If IssuerKey is being replaced by IssuerKeysDirectory, that has a very specific implication that the former shouldn't be set and if it is, the replacement of the deprecation will be handled by the function on line 606 and line 1491 shouldn't be set.

What happens when they are both set? Does it cause issues? If it can't be nicely replaced, then remove the replaced by part in param.yaml.

Reply:
The reason why I set both `IssuerKey` and `IssuerKeysDirectory` is migrating existing private key file. In the `migratePrivateKey` function, it first checks if there is an key at `IssuerKey`, if so, it will move the key to `IssuerKeysDirectory`. In `docs/parameters.yaml`, I tag `IssuerKey` with 'deprecated' and 'replacedby: IssuerKeysDirectory' to let user know this transition.